pages/linux/pacman4console.md

```diff
@@ -0,0 +1,20 @@
+# pacman4console
+
+> A text-based console game inspired by the original Pacman.
+> You play as Pacman (C); avoid the ghosts (&'s); collect all the pellets (*'s and .'s).
+
+- Start a game (at level 1 by default):
+
+`pacman4console`
+
+- Start a game on a certain level (there are nine official levels):
+
+`pacman4console --level={{#}}`
+
+- Start the pacman4console editor with a level text-file:
```

Comment:
I'm not sure about this example, as it uses a different command. tldr pages should only contain examples for a single specific command. Usually, we split multiple related commands into their own pages, including something liek this as the 2nd description line:

```
> See also: `other_command_name`.
```

What do you think? If we're going to keep this, perhaps this will read better?
```suggestion
- Start the pacman4console level editor, saving to a specified text file:
```

Reply:
I know ```pacman4consoleedit``` _is_ a different command, though I just thought that since it installs with ```pacman4console``` by default, and there's only one command for it, it might be worth bundling them (since AFAIK there's only one command for using ```pacman4consoleedit```. 

However, it definitely makes sense to separate them and have a ```> See also:```, since otherwise a ```tldr``` search for ```pacman4consoleedit``` would come out empty (though again, there's only one command for using it, so...)

Honestly, I don't know what to do about this.

Reply:
Hrm. Perhaps a 2nd opinion would be useful. @pxgamer / @mebeim / @agnivade, what do you think?

Reply:
We do have some commands where other related commands are used in the same page. But only when they are very closely related. This does seem to be the case.

@mtpiercey - btw, the command is wrongly spelled as `pacmanforconsoleedit`. Note the `for` instead of `4`.

Reply:
OK, I fixed the typo; thanks for noticing that.